[Bug] Frame Motors can't move Multipart blocks (Including Frames with tubes)

### Minecraft version

1.21.1

### CodeChickenLib version

4.6.1.524

### CBMultipart version

3.5.0.151

### ProjectRed version

4.22.0-beta+21

### What happened?

If you place a frame, and put something in it so it becomes a multipart block, frame motors become unable to move them.
Also, redstone breaks when interacting with the motor?
Here's a video of it:
https://www.youtube.com/watch?v=FIRyjePfi1E

### Crash log

Not a crash, but the error that appears in the log whenever I try to power the frame motor:
https://gist.github.com/DarianLStephens/f75121045c5ac0b1ca19f0ef6c52b322

### How do you make it happen again?

Place and power a Frame Motor
Place a frame
Put something inside that frame (Pneumatic Tube, Cover, etc)
Attempt to activate the Frame Motor

API jar is correctly included and loaded by Jar-in-Jar, but for whatever reason, its module is not able to access native Minecraft classes..

```
Caused by: java.lang.IllegalAccessError: class mrtjp.projectred.api.IExpansionAPI (in module mrtjp.ProjectRed.beta._21.api) cannot access class net.minecraft.resources.ResourceLocation (in module minecraft)
```

--- a/api/build.gradle
+++ b/api/build.gradle
@@ -8,3 +8,10 @@ neoForge {
 
 dependencies {
 }
+
+jar {
+    manifest {
+        // Required since API uses Minecraft classes
+        attributes 'FMLModType': 'GAMELIBRARY'
+    }
+}

--- a/core/build.gradle
+++ b/core/build.gradle
@@ -31,7 +31,7 @@ dependencies {
     accessTransformers "io.codechicken:CBMultipart:${mc_version}-${cbm_version}"
 
     // ProjectRed API
-    jarJar(implementation project(":api")) // Use JarJar to bake within Core jar
+    jarJar(api project(":api")) // Use JarJar to bake within Core jar
 
     // JEI
     compileOnly("mezz.jei:jei-${mc_version}-common-api:${jei_version}")

--- a/expansion/build.gradle
+++ b/expansion/build.gradle
@@ -24,6 +24,5 @@ dependencies {
     accessTransformers "io.codechicken:CodeChickenLib:${mc_version}-${ccl_version}"
     accessTransformers "io.codechicken:CBMultipart:${mc_version}-${cbm_version}"
 
-    implementation project(":api")
     implementation project(":core")
 }

--- a/exploration/build.gradle
+++ b/exploration/build.gradle
@@ -24,6 +24,5 @@ dependencies {
     accessTransformers "io.codechicken:CodeChickenLib:${mc_version}-${ccl_version}"
     accessTransformers "io.codechicken:CBMultipart:${mc_version}-${cbm_version}"
 
-    implementation project(":api")
     implementation project(":core")
 }

--- a/fabrication/build.gradle
+++ b/fabrication/build.gradle
@@ -24,7 +24,6 @@ dependencies {
     accessTransformers "io.codechicken:CodeChickenLib:${mc_version}-${ccl_version}"
     accessTransformers "io.codechicken:CBMultipart:${mc_version}-${cbm_version}"
 
-    implementation project(":api")
     implementation project(":core")
     implementation project(":integration")
     implementation project(":transmission")

--- a/gradle.properties
+++ b/gradle.properties
@@ -14,6 +14,6 @@ fabrication_version=0.1.0-alpha-19
 org.gradle.jvmargs=-Xmx4096M
 
 # The Minecraft version the Parchment version is for
-neogradle.subsystems.parchment.minecraftVersion=1.21.1
+neoForge.parchment.minecraftVersion=1.21.1
 # The version of the Parchment mappings
-neogradle.subsystems.parchment.mappingsVersion=2024.11.17
+neoForge.parchment.mappingsVersion=2024.11.17

--- a/illumination/build.gradle
+++ b/illumination/build.gradle
@@ -24,6 +24,5 @@ dependencies {
     accessTransformers "io.codechicken:CodeChickenLib:${mc_version}-${ccl_version}"
     accessTransformers "io.codechicken:CBMultipart:${mc_version}-${cbm_version}"
 
-    implementation project(":api")
     implementation project(":core")
 }

--- a/integration/build.gradle
+++ b/integration/build.gradle
@@ -24,6 +24,5 @@ dependencies {
     accessTransformers "io.codechicken:CodeChickenLib:${mc_version}-${ccl_version}"
     accessTransformers "io.codechicken:CBMultipart:${mc_version}-${cbm_version}"
 
-    implementation project(":api")
     implementation project(":core")
 }

--- a/transmission/build.gradle
+++ b/transmission/build.gradle
@@ -24,6 +24,5 @@ dependencies {
     accessTransformers "io.codechicken:CodeChickenLib:${mc_version}-${ccl_version}"
     accessTransformers "io.codechicken:CBMultipart:${mc_version}-${cbm_version}"
 
-    implementation project(":api")
     implementation project(":core")
 }

